./vcpkg broken on macOS 10.14.4

```
┌─[adam][hapkido][±][master ✓][~/vcpkg]
└─▪ ./vcpkg update
dyld: Symbol not found: __ZNKSt19__codecvt_utf8_baseIwE10do_unshiftER11__mbstate_tPcS3_RS3_
  Referenced from: /Users/adam/vcpkg/./vcpkg
  Expected in: /usr/lib/libstdc++.6.dylib
 in /Users/adam/vcpkg/./vcpkg
Abort trap: 6
```
This is after I did a `git pull`:

```
┌─[adam][hapkido][±][master ✓][~/vcpkg]
└─▪ git pull
remote: Enumerating objects: 184, done.
remote: Counting objects: 100% (184/184), done.
remote: Total 453 (delta 184), reused 184 (delta 184), pack-reused 269
Receiving objects: 100% (453/453), 122.39 KiB | 12.24 MiB/s, done.
Resolving deltas: 100% (221/221), completed with 134 local objects.
From https://github.com/Microsoft/vcpkg
   93dedf0a..14bfca23  master     -> origin/master
Updating 93dedf0a..14bfca23
Fast-forward
 docs/about/faq.md                                  |   2 +-
 docs/maintainers/control-files.md                  |  91 +++-
 ports/abseil/CONTROL                               |   2 +-
 ports/abseil/portfile.cmake                        |   4 +-
 ports/ace/CONTROL                                  |  10 +
 ports/ace/portfile.cmake                           |  24 +-
 ports/assimp/CONTROL                               |   2 +-
 ports/aws-sdk-cpp/CONTROL                          |   2 +-
 ports/aws-sdk-cpp/portfile.cmake                   |   4 +-
 ports/breakpad/CONTROL                             |   2 +-
 ports/breakpad/portfile.cmake                      |   4 +-
 ports/butteraugli/CONTROL                          |   2 +-
 ports/butteraugli/portfile.cmake                   |   4 +-
 ports/caf/CONTROL                                  |   2 +-
 ports/caf/portfile.cmake                           |   4 +-
 ports/cartographer/CONTROL                         |   2 +-
 ports/catch2/CONTROL                               |   2 +-
 ports/catch2/portfile.cmake                        |   4 +-
 ports/cgltf/CONTROL                                |   3 +
 ports/cgltf/portfile.cmake                         |  16 +
 ports/cimg/CONTROL                                 |   2 +-
 ports/cimg/portfile.cmake                          |   4 +-
 ports/cppgraphqlgen/CONTROL                        |   2 +-
 ports/cppgraphqlgen/portfile.cmake                 |   4 +-
 ports/cpprestsdk/CONTROL                           |   4 +-
 ports/cpr/001-cpr-config.patch                     |  94 ++++
 ports/cpr/CONTROL                                  |   2 +-
 ports/cpr/cprConfig.cmake                          |  27 +
 ports/cpr/portfile.cmake                           |   8 +-
 ports/directxmesh/CONTROL                          |   2 +-
 ports/directxmesh/portfile.cmake                   |   4 +-
 ports/directxtex/CONTROL                           |   2 +-
 ports/directxtex/portfile.cmake                    |   4 +-
 ports/directxtk/CONTROL                            |   2 +-
 ports/directxtk/portfile.cmake                     |   4 +-
 ports/dlib/CONTROL                                 |   2 +-
 ports/dlib/portfile.cmake                          |   2 +-
 ports/doctest/CONTROL                              |   2 +-
 ports/doctest/portfile.cmake                       |   4 +-
 ports/eastl/CONTROL                                |   2 +-
 ports/eastl/fixchar8_t.patch                       |  13 +
 ports/eastl/portfile.cmake                         |   5 +-
 ports/ecsutil/CONTROL                              |   2 +-
 ports/ecsutil/portfile.cmake                       |   4 +-
 ports/ensmallen/CONTROL                            |   2 +-
 ports/ensmallen/portfile.cmake                     |   4 +-
 ports/forest/CONTROL                               |   2 +-
 ports/forest/portfile.cmake                        |   4 +-
 ports/freeglut/CONTROL                             |   2 +-
 ports/freeglut/macOS_Xquartz.patch                 |  21 +
 ports/freeglut/portfile.cmake                      |  29 +-
 .../use_targets_to_export_x11_dependency.patch     |  29 +
 ports/freeglut/vcpkg-cmake-wrapper.cmake           |  10 +
 ports/freeimage/portfile.cmake                     |  35 +-
 .../use-typedef-as-already-declared.patch          |  15 +
 ports/gdal/0003-Fix-std-fabs.patch                 |  10 +
 ports/gdal/CONTROL                                 |   2 +-
 ports/gdal/portfile.cmake                          | 582 ++++++++++++---------
 ports/gdcm/CONTROL                                 |   4 +
 ports/{gdcm2 => gdcm}/find-openjpeg.patch          |   0
 ports/gdcm/fix-share-path.patch                    |  32 ++
 ports/gdcm/portfile.cmake                          |  49 ++
 ports/gdcm2/CONTROL                                |   6 +-
 ports/gdcm2/portfile.cmake                         |  47 +-
 ports/geogram/CONTROL                              |   4 +-
 ports/geogram/portfile.cmake                       |  24 +-
 ports/giflib/portfile.cmake                        |   8 +-
 ports/google-cloud-cpp/CONTROL                     |   2 +-
 ports/google-cloud-cpp/cmake-libcurl-target.patch  | 144 -----
 ports/google-cloud-cpp/portfile.cmake              |   5 +-
 ports/graphqlparser/CONTROL                        |   2 +-
 ports/graphqlparser/portfile.cmake                 |   4 +-
 ports/graphqlparser/static-compile-fix.patch       |  16 +
 ports/grpc/portfile.cmake                          |   2 +-
 ports/guetzli/CONTROL                              |   2 +-
 ports/guetzli/butteraugli.patch                    |  13 +
 ports/guetzli/portfile.cmake                       |   1 +
 ports/harfbuzz/CONTROL                             |   2 +-
 ports/harfbuzz/portfile.cmake                      |   4 +-
 ports/imgui/CONTROL                                |   2 +-
 ports/imgui/portfile.cmake                         |   4 +-
 ports/itk/CONTROL                                  |   2 +-
 ports/itk/hdf5_config_mode_find_package.patch      |  68 ---
 ports/itk/portfile.cmake                           |  15 +-
 ports/jsonnet/001-enable-msvc.patch                |  78 +++
 ports/jsonnet/CMakeLists.txt                       |  49 --
 ports/jsonnet/CONTROL                              |   6 +-
 ports/jsonnet/portfile.cmake                       |  22 +-
 ports/libpng/CONTROL                               |   2 +-
 ports/libpng/portfile.cmake                        |  10 +-
 ports/libpng/skip-install-symlink.patch            |  22 -
 ports/libpng/usage                                 |   4 -
 ports/libwebp/0001-build-fixes.patch               |   4 +-
 ports/libwebp/0003-remove-missing-symbol.patch     |  13 +
 .../libwebp/0004-add-missing-linked-library.patch  |  15 +
 ports/libwebp/CONTROL                              |   4 +-
 ports/libwebp/portfile.cmake                       |  65 ++-
 ports/loguru/CONTROL                               |   4 +
 ports/loguru/copyright                             |   3 +
 ports/loguru/portfile.cmake                        |  12 +
 ports/luasocket/CONTROL                            |   2 +-
 ports/luasocket/portfile.cmake                     |   4 +-
 ports/lz4/CONTROL                                  |   2 +-
 ports/lz4/portfile.cmake                           |   4 +-
 ports/minhook/CONTROL                              |   3 +
 ports/minhook/install-destination.patch            |  22 +
 ports/minhook/portfile.cmake                       |  49 ++
 ports/moos-core/CONTROL                            |   7 +-
 ports/moos-core/portfile.cmake                     |  39 +-
 ports/nana/CONTROL                                 |   2 +-
 ports/nana/portfile.cmake                          |   4 +-
 ports/openblas/CONTROL                             |   2 +-
 ports/openblas/portfile.cmake                      |   4 +-
 ports/opencv/CONTROL                               |   2 +-
 ports/openjpeg/CONTROL                             |   2 +-
 ports/openjpeg/portfile.cmake                      |   2 -
 ports/osg/CONTROL                                  |   2 +-
 ports/osg/portfile.cmake                           |  16 +-
 ports/qt5-3d/CONTROL                               |   2 +-
 ports/qt5-3d/portfile.cmake                        |   2 +-
 ports/qt5-activeqt/CONTROL                         |   2 +-
 ports/qt5-activeqt/portfile.cmake                  |   2 +-
 ports/qt5-base/CONTROL                             |   8 +-
 ports/qt5-base/fix-gui-configure-json.patch        |  40 --
 ports/qt5-base/portfile.cmake                      |  10 +-
 ports/qt5-base/vcpkg-cmake-wrapper.cmake           |  26 +-
 ports/qt5-charts/CONTROL                           |   2 +-
 ports/qt5-charts/portfile.cmake                    |   2 +-
 ports/qt5-connectivity/CONTROL                     |   2 +-
 ports/qt5-connectivity/portfile.cmake              |   2 +-
 ports/qt5-datavis3d/CONTROL                        |   2 +-
 ports/qt5-datavis3d/portfile.cmake                 |   2 +-
 ports/qt5-declarative/CONTROL                      |   2 +-
 ports/qt5-declarative/portfile.cmake               |   9 +-
 ports/qt5-gamepad/CONTROL                          |   2 +-
 ports/qt5-gamepad/portfile.cmake                   |   2 +-
 ports/qt5-graphicaleffects/CONTROL                 |   2 +-
 ports/qt5-graphicaleffects/portfile.cmake          |   2 +-
 ports/qt5-imageformats/CONTROL                     |   2 +-
 ports/qt5-imageformats/portfile.cmake              |   2 +-
 ports/qt5-location/CONTROL                         |   2 +-
 ports/qt5-location/portfile.cmake                  |   2 +-
 ports/qt5-macextras/CONTROL                        |   2 +-
 ports/qt5-macextras/portfile.cmake                 |   2 +-
 ports/qt5-modularscripts/CONTROL                   |   6 +-
 ports/qt5-modularscripts/qt_modular_library.cmake  |   2 +-
 ports/qt5-mqtt/CONTROL                             |   3 +-
 ports/qt5-mqtt/portfile.cmake                      |   4 +-
 ports/qt5-multimedia/CONTROL                       |   2 +-
 ports/qt5-multimedia/portfile.cmake                |   2 +-
 ports/qt5-networkauth/CONTROL                      |   2 +-
 ports/qt5-networkauth/portfile.cmake               |   2 +-
 ports/qt5-purchasing/CONTROL                       |   2 +-
 ports/qt5-purchasing/portfile.cmake                |   2 +-
 ports/qt5-quickcontrols/CONTROL                    |   2 +-
 ports/qt5-quickcontrols/portfile.cmake             |   2 +-
 ports/qt5-quickcontrols2/CONTROL                   |   2 +-
 ports/qt5-quickcontrols2/portfile.cmake            |   2 +-
 ports/qt5-remoteobjects/CONTROL                    |   2 +-
 ports/qt5-remoteobjects/portfile.cmake             |   2 +-
 ports/qt5-script/CONTROL                           |   2 +-
 ports/qt5-script/portfile.cmake                    |   2 +-
 ports/qt5-scxml/CONTROL                            |   2 +-
 ports/qt5-scxml/portfile.cmake                     |   2 +-
 ports/qt5-sensors/CONTROL                          |   2 +-
 ports/qt5-sensors/portfile.cmake                   |   2 +-
 ports/qt5-serialport/CONTROL                       |   2 +-
 ports/qt5-serialport/portfile.cmake                |   2 +-
 ports/qt5-speech/CONTROL                           |   2 +-
 ports/qt5-speech/portfile.cmake                    |   2 +-
 ports/qt5-svg/CONTROL                              |   2 +-
 ports/qt5-svg/portfile.cmake                       |   2 +-
 ports/qt5-tools/CONTROL                            |   2 +-
 ports/qt5-tools/portfile.cmake                     |   2 +-
 ports/qt5-virtualkeyboard/CONTROL                  |   2 +-
 ports/qt5-virtualkeyboard/portfile.cmake           |   2 +-
 ports/qt5-webchannel/CONTROL                       |   2 +-
 ports/qt5-webchannel/portfile.cmake                |   2 +-
 ports/qt5-websockets/CONTROL                       |   2 +-
 ports/qt5-websockets/portfile.cmake                |   2 +-
 ports/qt5-webview/CONTROL                          |   2 +-
 ports/qt5-webview/portfile.cmake                   |   2 +-
 ports/qt5-winextras/CONTROL                        |   2 +-
 ports/qt5-winextras/portfile.cmake                 |   2 +-
 ports/qt5-xmlpatterns/CONTROL                      |   2 +-
 ports/qt5-xmlpatterns/portfile.cmake               |   2 +-
 ports/qt5/CONTROL                                  |   2 +-
 ports/re2/CONTROL                                  |   2 +-
 ports/re2/portfile.cmake                           |   4 +-
 ports/rocksdb/CONTROL                              |   2 +-
 ports/rocksdb/portfile.cmake                       |   4 +-
 ports/rs-core-lib/CONTROL                          |   2 +-
 ports/rs-core-lib/portfile.cmake                   |   4 +-
 ...Deduce-reference-in-boost-fusion-for_each.patch |  75 +++
 .../soci/0002-Find-PostgreSQL-debug-library.patch  |  62 +++
 ports/soci/CONTROL                                 |   2 +-
 ports/soci/portfile.cmake                          |   2 +
 ports/stb/CONTROL                                  |   2 +-
 ports/stb/portfile.cmake                           |   4 +-
 ports/taocpp-json/CONTROL                          |   2 +-
 ports/taocpp-json/portfile.cmake                   |   4 +-
 ports/tbb/CONTROL                                  |   2 +-
 ports/tbb/portfile.cmake                           |   4 +-
 ports/thrift/CONTROL                               |   2 +-
 ports/thrift/portfile.cmake                        |   4 +-
 ports/unicorn-lib/CONTROL                          |   2 +-
 ports/unicorn-lib/portfile.cmake                   |   4 +-
 ports/uriparser/CONTROL                            |   5 +-
 ports/uriparser/portfile.cmake                     |  62 ++-
 ports/valijson/CONTROL                             |   3 +
 ports/valijson/fix-nlohmann-json.patch             |  26 +
 ports/valijson/fix-optional.patch                  |  13 +
 ports/valijson/fix-picojson.patch                  |  26 +
 ports/valijson/portfile.cmake                      |  28 +
 ports/visit-struct/CMakeLists.txt                  |  17 +
 ports/visit-struct/CONTROL                         |   3 +
 ports/visit-struct/portfile.cmake                  |  30 ++
 ports/vtk-dicom/CONTROL                            |   2 +-
 ports/wil/CONTROL                                  |   3 +
 ports/wil/portfile.cmake                           |  13 +
 ports/x264/CONTROL                                 |   2 +-
 ports/x264/portfile.cmake                          |  20 +-
 ports/x264/uwp-cflags.patch                        |  24 +-
 ports/xsimd/CONTROL                                |   2 +-
 ports/xsimd/portfile.cmake                         |   4 +-
 ports/zeromq/CONTROL                               |   2 +-
 ports/zeromq/portfile.cmake                        |   4 +-
 ports/zxing-cpp/0001-opencv4-compat.patch          |  30 ++
 ports/zxing-cpp/CONTROL                            |   2 +-
 ports/zxing-cpp/portfile.cmake                     |   2 +
 230 files changed, 1808 insertions(+), 1009 deletions(-)
 create mode 100644 ports/cgltf/CONTROL
 create mode 100644 ports/cgltf/portfile.cmake
 create mode 100644 ports/cpr/001-cpr-config.patch
 create mode 100644 ports/cpr/cprConfig.cmake
 create mode 100644 ports/eastl/fixchar8_t.patch
 create mode 100644 ports/freeglut/macOS_Xquartz.patch
 create mode 100644 ports/freeglut/use_targets_to_export_x11_dependency.patch
 create mode 100644 ports/freeglut/vcpkg-cmake-wrapper.cmake
 create mode 100644 ports/freeimage/use-typedef-as-already-declared.patch
 create mode 100644 ports/gdal/0003-Fix-std-fabs.patch
 create mode 100644 ports/gdcm/CONTROL
 rename ports/{gdcm2 => gdcm}/find-openjpeg.patch (100%)
 create mode 100644 ports/gdcm/fix-share-path.patch
 create mode 100644 ports/gdcm/portfile.cmake
 delete mode 100644 ports/google-cloud-cpp/cmake-libcurl-target.patch
 create mode 100644 ports/graphqlparser/static-compile-fix.patch
 create mode 100644 ports/guetzli/butteraugli.patch
 delete mode 100644 ports/itk/hdf5_config_mode_find_package.patch
 create mode 100644 ports/jsonnet/001-enable-msvc.patch
 delete mode 100644 ports/jsonnet/CMakeLists.txt
 delete mode 100644 ports/libpng/skip-install-symlink.patch
 delete mode 100644 ports/libpng/usage
 create mode 100644 ports/libwebp/0003-remove-missing-symbol.patch
 create mode 100644 ports/libwebp/0004-add-missing-linked-library.patch
 create mode 100644 ports/loguru/CONTROL
 create mode 100644 ports/loguru/copyright
 create mode 100644 ports/loguru/portfile.cmake
 create mode 100644 ports/minhook/CONTROL
 create mode 100644 ports/minhook/install-destination.patch
 create mode 100644 ports/minhook/portfile.cmake
 delete mode 100644 ports/qt5-base/fix-gui-configure-json.patch
 create mode 100644 ports/soci/0001-Deduce-reference-in-boost-fusion-for_each.patch
 create mode 100644 ports/soci/0002-Find-PostgreSQL-debug-library.patch
 create mode 100644 ports/valijson/CONTROL
 create mode 100644 ports/valijson/fix-nlohmann-json.patch
 create mode 100644 ports/valijson/fix-optional.patch
 create mode 100644 ports/valijson/fix-picojson.patch
 create mode 100644 ports/valijson/portfile.cmake
 create mode 100644 ports/visit-struct/CMakeLists.txt
 create mode 100644 ports/visit-struct/CONTROL
 create mode 100644 ports/visit-struct/portfile.cmake
 create mode 100644 ports/wil/CONTROL
 create mode 100644 ports/wil/portfile.cmake
 create mode 100644 ports/zxing-cpp/0001-opencv4-compat.patch
```

I deleted it entirely and attempted to reinstall:

```
┌─[adam][hapkido][~]
└─▪ rm -rf vcpkg/
┌─[adam][hapkido][~]
└─▪ git clone https://github.com/microsoft/vcpkg.git
Cloning into 'vcpkg'...
remote: Enumerating objects: 54194, done.
remote: Total 54194 (delta 0), reused 0 (delta 0), pack-reused 54194
Receiving objects: 100% (54194/54194), 12.72 MiB | 4.20 MiB/s, done.
Resolving deltas: 100% (34576/34576), done.
┌─[adam][hapkido][~]
└─▪ cd vcpkg/
┌─[adam][hapkido][±][master ✓][~/vcpkg]
└─▪ ./bootstrap-vcpkg.sh 
Downloading cmake...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   621    0   621    0     0   2419      0 --:--:-- --:--:-- --:--:--  2416
100 31.9M  100 31.9M    0     0  3028k      0  0:00:10  0:00:10 --:--:-- 3153k
Downloading cmake... done.
Extracting cmake...
Extracting cmake... done.
Downloading ninja...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   602    0   602    0     0   1974      0 --:--:-- --:--:-- --:--:--  1980
100 77284  100 77284    0     0  87557      0 --:--:-- --:--:-- --:--:--  157k
Downloading ninja... done.
Extracting ninja...
Extracting ninja... done.
/Users/adam/vcpkg/scripts/bootstrap.sh: line 217: [: Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10: integer expression expected
-- The C compiler identification is AppleClang 10.0.1.10010046
-- The CXX compiler identification is AppleClang 10.0.1.10010046
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/g++
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:12 (message):
  Building the vcpkg tool requires support for the C++ Filesystem TS.

  Apple clang versions 9 and below do not have support for it.

  Please install gcc6 or newer from homebrew (brew install gcc6).

  If you would like to try anyway, pass --allowAppleClang to bootstrap.sh.


-- Configuring incomplete, errors occurred!
See also "/Users/adam/vcpkg/toolsrc/build.rel/CMakeFiles/CMakeOutput.log".
```

I have gcc 9.1.0 installed, and `--allowAppleClang` doesn't work.

```
┌─[adam][hapkido][±][master ✓][~/vcpkg]
└─▪ brew info gcc
gcc: stable 9.1.0 (bottled), HEAD
GNU compiler collection
https://gcc.gnu.org/
/usr/local/Cellar/gcc/9.1.0 (1,461 files, 287MB) *
  Poured from bottle on 2019-05-08 at 19:18:06
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/gcc.rb
==> Dependencies
Required: gmp ✔, isl ✔, libmpc ✔, mpfr ✔
==> Options
--HEAD
	Install HEAD version
==> Analytics
install: 124,395 (30 days), 308,503 (90 days), 829,293 (365 days)
install_on_request: 58,405 (30 days), 145,659 (90 days), 401,065 (365 days)
build_error: 0 (30 days)
┌─[adam][hapkido][±][master ✓][~/vcpkg]
└─▪ ./bootstrap-vcpkg.sh --allowAppleClang
/Users/adam/vcpkg/scripts/bootstrap.sh: line 217: [: Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10: integer expression expected
-- The C compiler identification is AppleClang 10.0.1.10010046
-- The CXX compiler identification is AppleClang 10.0.1.10010046
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/g++
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test USES_LIBSTDCXX
-- Performing Test USES_LIBSTDCXX - Failed
-- Performing Test USES_LIBCXX
-- Performing Test USES_LIBCXX - Success
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE  
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/adam/vcpkg/toolsrc/build.rel
[1/64] Building CXX object CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o
FAILED: CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o 
/Library/Developer/CommandLineTools/usr/bin/g++  -DDISABLE_METRICS=0 -I../include -O3 -DNDEBUG -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk   -std=c++1z -MD -MT CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o -MF CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o.d -o CMakeFiles/vcpkg.dir/src/vcpkg.cpp.o -c ../src/vcpkg.cpp
In file included from ../src/vcpkg.cpp:24:
../include/vcpkg/base/files.h:8:10: fatal error: 'experimental/filesystem' file not found
#include <experimental/filesystem>
         ^~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/scripts/bootstrap.sh
+++ b/scripts/bootstrap.sh
@@ -203,7 +203,9 @@ selectCXX()
 
     if [ "x$CXX" = "x" ]; then
         CXX=g++
-        if which g++-8 >/dev/null 2>&1; then
+        if which g++-9 >/dev/null 2>&1; then
+            CXX=g++-9
+        elif which g++-8 >/dev/null 2>&1; then
             CXX=g++-8
         elif which g++-7 >/dev/null 2>&1; then
             CXX=g++-7

